source/AFRL_CRRES_models.f

```diff
@@ -186,8 +186,8 @@ SUBROUTINE get_crres_flux(ntmax,whichm,whatf,nene,
 c
        INTEGER*4   Ne,Nl,Nbb0,ind
 c
-       REAL*8      energy(2,25)
-       REAL*8      flux(ntime_max,25),BBo(ntime_max),L(ntime_max)
+       REAL*8      energy(2,nene)
+       REAL*8      flux(ntime_max,nene),BBo(ntime_max),L(ntime_max)
```

Comment:
This should be updated in the HTML docs.

Reply:
I've updated the documentation to reflect the fact that the number of energies is not limited anymore. Note that all these modifications are supposed to be backward-compatible with legacy usages.